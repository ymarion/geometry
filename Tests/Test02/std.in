LOAD model.txt
# Checking if the file has been loaded
LIST
#
EXIT
