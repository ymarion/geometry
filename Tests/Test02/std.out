# The loading OK message
OK
# Four insertions
OK
OK
OK
OK
# The result of LIST should be the one of the file
C C1 0 0 2
L L1 0 0 20 20
PL PL1 10 10 10 15 15 15 15 10
R R1 0 0 20 20
